extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -269,8 +269,18 @@ export class FileConfigPage extends ImportPage {
 		let connectionUri = await azdata.connection.getUriForConnection(this.model.server.connectionId);
 		let queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>(this.model.server.providerName, azdata.DataProviderType.QueryProvider);
 
-		const escapedQuotedDb = this.databaseDropdown.value ? `[${(<azdata.CategoryValue>this.databaseDropdown.value).name.replace(/]/g, ']]')}].` : '';
-		const query = `SELECT name FROM ${escapedQuotedDb}sys.schemas`;
+		let currentServer = await azdata.connection.getServerInfo(this.model.server.connectionId);
+
+		let query: string;
+
+		// Azure SQL server doesn't support use statements.
```

Comment:
So first off - this isn't a use statement so the comment doesn't really make sense here. 

Second - Azure DOES support db-qualified names as long as you're connected to the DB you're trying to query. So what that implies is that if the user uses a connection to a user-DB and then selects a different DB from the dropdown the query fails (as expected).

What we really should be doing is establishing a new connection to that specific database. Because otherwise we'll get the wrong list of schemas because the connection is always going to be to the original DB.

Reply:
I have used the change database API to resolve this issue. Please check the updated code